www/js/lib/settingsStore.js

```diff
@@ -98,56 +98,66 @@ define([], function () {
    * @param {String} object Optional name of the object to disable or delete ('cookie', 'localStorage', 'cacheAPI')
    */
   function reset(object) {
-    // If no specific object was specified, we are doing a general reset, so ask user for confirmation
-    if (!object && !confirm('WARNING: This will reset the app to a freshly installed state, deleting all app caches and settings!')) return;
-    
-    // 1. Clear any cookie entries
-    if (!object || object === 'cookie') {
-      var regexpCookieKeys = /(?:^|;)\s*([^=]+)=([^;]*)/ig;
-      var currentCookie = document.cookie;
-      var foundCrumb = false;
-      var cookieCrumb = regexpCookieKeys.exec(currentCookie);
-      while (cookieCrumb !== null) {
-        // DEV: Note that we don't use the keyPrefix in legacy cookie support
-        foundCrumb = true;
-        // This expiry date will cause the browser to delete the cookie crumb on next page refresh
-        document.cookie = cookieCrumb[1] + '=;expires=Thu, 21 Sep 1979 00:00:01 UTC;';
-        cookieCrumb = regexpCookieKeys.exec(currentCookie);
+    const performReset = function () {
```

Comment:
I don't know if we should set a function as a const . It does work in IE11, but I wonder if it will work in all contexts. Maybe best just to use var?

Reply:
Okay